src/main/resources/entity-types/inventory/simple_instance.json5

```diff
@@ -416,17 +416,20 @@
       queryable: true,
       essential: true,
       visibleByDefault: false,
-      valueGetter: "(SELECT array_agg(a.jsonb ->> 'name') FILTER (WHERE (a.jsonb ->> 'name') IS NOT NULL) AS array_agg FROM jsonb_array_elements((:sourceAlias.jsonb -> 'contributors') -> 'contributorTypeId') AS record(value) JOIN src_inventory_contributor_type a ON (record.value::text) = a.id::text)",
-      filterValueGetter: "( SELECT array_agg(lower(a.jsonb ->> 'name'::text))  FILTER (WHERE (a.jsonb ->> 'name') IS NOT NULL) AS array_agg FROM jsonb_array_elements_text((:sourceAlias.jsonb -> 'contributors'::text) ->> 'contributorTypeId'::text) record(value) JOIN src_inventory_contributor_type a ON (record.value::text) = a.id::text)",
-      valueFunction: 'lower(:value)',
+      valueGetter:  "(SELECT array_agg(class_type.jsonb ->> 'name') \
```

Comment:
can we change class_type to contributor_type, just to make it less confusing when dealing with these in the future?